rb/spec/integration/selenium/webdriver/element_spec.rb

```diff
@@ -94,6 +94,11 @@ module WebDriver
         expect(driver.find_element(id: 'withText').attribute('nonexistent')).to be_nil
       end
 
+      it 'should get property value' do
```

Comment:
As for spec guards, you're right, there shouldn't be any need for this with the backward compatible feature. Even for Edge... Since Edge is only half supporting W3C so far, I created [this workaround](https://github.com/SeleniumHQ/selenium/blob/master/rb/lib/selenium/webdriver/edge/legacy_support.rb). We can actually make this workaround less complicated now that as of 3.0.1 Ruby sends JWP commands to Remote always.